src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -145,8 +145,23 @@ void fillInnerParts() {
         StatusBarFooter statusBarFooter = new StatusBarFooter(logic.getTeamListFilePath());
         statusbarPlaceholder.getChildren().add(statusBarFooter.getRoot());
 
-        CommandBox commandBox = new CommandBox(this::executeCommand);
+        commandBox = new CommandBox(this::executeCommand);
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
+        setCommandNavigationHandler();
+    }
+
+    private void setCommandNavigationHandler() {
+        this.commandBoxPlaceholder.addEventFilter(KeyEvent.KEY_PRESSED, event -> {
+            if (event.getCode() == KeyCode.UP) {
+                System.out.println("Up Key Pressed");
```

Comment:
Can omit this, CLI is already very verbose

Reply:
Yup. Removed them. Will remove the debug statements in the other components I did in subsequent PRs. 👍